src/addremove.js

```diff
@@ -0,0 +1,83 @@
+const Task = [] || JSON.parse(localStorage.getItem('Task'));
+
+const TaskList = document.querySelector('.to-do-list');
+
+const createElement = (element) => {
+  const InsertedDiv = document.createElement('div');
+  const InsertedCheckBox = document.createElement('input');
+  const toDoTask = document.createElement('input');
+  const deleteButton = document.createElement('button');
+
+  deleteButton.classList.add('remove-task');
+  deleteButton.innerHTML = 'Remove';
+
+  InsertedDiv.classList.add('dynamic-Elements');
+
+  InsertedCheckBox.type = 'checkbox';
+  InsertedCheckBox.checked = element.complete;
+
+  toDoTask.value = element.description;
+  toDoTask.classList.add('task-layout');
+
+  InsertedDiv.setAttribute('index_id', element.index);
+  InsertedDiv.append(InsertedCheckBox, toDoTask, deleteButton);
+
+  TaskList.appendChild(InsertedDiv);
+};
+
+const addTask = (Task) => {
+  Task.forEach((element) => {
+    createElement(element);
+  });
+};
+
+const setLocalStorage = (Task) => {
+  window.localStorage.setItem('Task', JSON.stringify(Task));
+};
+
+const getLocalStorage = () => {
+  const theTask = window.localStorage.getItem('Task');
+
+  if (theTask !== null) {
+    addTask(JSON.parse(theTask));
+  }
+
+  return theTask;
+};
+
+const addToDoTask = (value) => {
+  const task = {
+    description: value,
+    completed: false,
+    index: Task.length + 1,
+  };
+
+  Task.push(task);
+  setLocalStorage(Task);
+  createElement(task);
+};
+
+const deleteToDoTask = (ID) => {
+  const theTask = JSON.parse(localStorage.getItem('Task'));
+  theTask.splice(ID - 1, 1);
+  theTask.forEach((element, index) => {
+    element.index = index + 1;
+  });
+  setLocalStorage(theTask);
+
+  return theTask;
+};
```

Comment:
- [ ] Upon removing one item from the list and adding another item immediately after that followed by reloading, only the last item is visible and local storage also holds only this last added item. Consider fixing it. If you don't understand what I am saying, contact me on slack.